plugins/by-name/leetcode/default.nix

```
@@ -0,0 +1,94 @@
+{ lib, ... }:
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "leetcode";
+  package = "leetcode-nvim"; # TODO replace
+
+  maintainers = [ lib.maintainers.Fovir ];
+
+  settingsExample = {
+    arg = "leetcode.nvim";
+
+    lang = "cpp";
+
+    cn = {
+      enabled = false;
+      translator = true;
+      translate_problems = true;
+    };
+
+    storage = {
+      home = ''vim.fn.stdpath("data") .. "/leetcode"'';
+      cache = ''vim.fn.stdpath("cache") .. "/leetcode"'';
+    };
+
+    plugins = {
+      non_standalone = false;
+    };
+
+    logging = true;
+
+    injector = { };
+
+    cache = {
+      update_interval = 60 * 60 * 24 * 7;
```

```suggestion
      update_interval = lib.nixvim.nestedLiteralLua "60 * 60 * 24 * 7";
```
Otherwise, the documentation will just show the final value. (which is maybe what you wanted?